services/retrieve_and_rank/v1.js

```diff
@@ -246,6 +288,36 @@ module.exports = function (RED) {
       });
     },interval);
   }
-};
 
 
+    function checkClusterStatus(retrieve_and_rank,msg,node,cluster_id) {
+      var interval = 10000;
+      //Loop and check the status of the ranker
+      var statusInterval = setInterval(function(){
+        var params = {
+          cluster_id: cluster_id
+        };
+        retrieve_and_rank.pollCluster(params, function(err, res) {
+            if (err) {
+              node.status({});
+              var message = "Cluster creation state error: "+err.error;
+              clearInterval(statusInterval);
+              return node.error(message, msg);
+            } else {
+              switch (res.status) {
+                case 'Available':
+                  node.status({fill:"green",shape:"ring",text:"Cluster available"});
+                  return clearInterval(statusInterval);
+                  break;
+
+                default:
+                  node.status({});
+                  var message = "Cluster status: "+res.status+", description: "+res.status_description;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2404865885/issues/source?bid=3225115&fileBranchId=3233374#l315)